libraries/AP_Logger/LogStructure.h

```diff
@@ -618,6 +618,7 @@ struct PACKED log_RFND {
     uint64_t time_us;
     uint8_t instance;
     uint16_t dist;
+    uint16_t snr;
```

Comment:
Could you also please update
```
    { LOG_RFND_MSG, sizeof(log_RFND), \      "RFND", "QBCBB", "TimeUS,Instance,Dist,Stat,Orient", "s#m--", "F-B--" }, \
```
to include snr? L1408 of this file

Reply:
Done. (It was line ~1946 in this branch)

Reply:
Thank you! I was looking to apply this commit to our cube firmware and must have been looking at the line numbers in there 🙈